### PR TITLE
CI handbook outline

### PR DESCRIPTION
**Pull request by jzwart**
# Summary 
This PR adds in a list of topics to cover in the cyberinfrastructure (CI) handbook - this is by no means comprehensive but it is based on most common themes that came up during the 2024 workshop and expert opinion of what's important to have in there. Most article topics are placeholders - I did start to flesh out the model standards output article, summarizing the model standards from Dietze et al. 2023. 

# Review needed 
- Let me know if you think the proposed topics seem reasonable to include in the handbook or if they need to be changed. We can propose other topics to include as well. 
- Is the model standards output article an appropriate level of information for the handbook? more information needed? should we include examples? 

Your can preview proposed changes following instructions here https://projects.ecoforecast.org/efi-ci-workshop-2024/#contributing

### CODE REVIEW

**Comment by jzwart** [comment]
@cboettig @mdietze @rqthomas , I summarized the forecast output standards document here https://github.com/eco4cast/efi-ci-workshop-2024/pull/15/files#diff-02acbe83e216a54b48103001bafe44c35d3fe3935b6f53a45f5561953595ea2d ; let us know if you think this is an appropriate level of detail to include or if more is needed, etc...

File: ci_handbook/model-output-standards.qmd [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| *parameter*          | **REQUIRED**. For ensemble predictions, specify integers from `1` to `Ne` (where `Ne` represents the total size of the ensemble). For named distributions, indicate the parameter or statistic being specified (e.g., `mean`, `sigma`).                                                                                                                                                                                                                                         |
```
Comment (rqthomas) [inline ci_handbook/model-output-standards.qmd:42]: change to (e.g., a normal distribution would have  `mean` and `sd` as the values in the parameter column)

File: ci_handbook/model-output-standards.qmd [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: "{{< fa arrow-right >}} Model Output Standards"
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    toc-position: left
+reading-time: true
+image: "../images/Lego_dimensions.svg.png"
```
Comment (sea-scope) [inline ci_handbook/model-output-standards.qmd:9]: I think this is the wrong image, you want the honeycomb here. I'm seeing the lego pieces which should be under 'modularity'

Comment (sea-scope) [reply ci_handbook/model-output-standards.qmd:9]: /images/hive.jpg

File: ci_handbook/modularity.qmd [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: "{{< fa arrow-right >}} Modularity"
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    toc-position: left
+reading-time: true
+image: "../images/hive.jpg"
```
Comment (sea-scope) [inline ci_handbook/modularity.qmd:9]: Should be showing lego pieces
/images/Lego_dimensions.svg.png"

File: ci_handbook/data_format.qmd [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: "{{< fa arrow-right >}} Data Format"
```
Comment (sea-scope) [inline ci_handbook/data_format.qmd:2]: 'Data Standards' may  be a more recognizable heading for this section

File: ci_handbook/index.qmd [hunk 5]
```diff
@@ -8,8 +8,19 @@ format:
     comments:
       utterances:
          repo: eco4cast/efi-ci-workshop-2024
+listing: 
+  contents:
```
Comment (sea-scope) [inline ci_handbook/index.qmd:12]: In the meeting we talked about an 'Examples' section, or would that be included under workflows?

File: ci_handbook/continuous_integration.qmd [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+title: "{{< fa arrow-right >}} Continuous Integration & Delivery"
```
Comment (sea-scope) [inline ci_handbook/continuous_integration.qmd:2]: Is this another way of saying 'Ecological Forecast Operationalization'
I don't know what is more commonly understood, but maybe 'Continuous Data Integration & Forecast Delivery' would be more explicit

**Review by sea-scope — APPROVED** [review]
Just a few minor comments.

File: ci_handbook/model-output-standards.qmd (hunk 2 above)
Comment (jzwart) [reply ci_handbook/model-output-standards.qmd:9]: I do want the lego pieces as the precise dimensions of the lego pieces are required to build something.

File: ci_handbook/modularity.qmd (hunk 3 above)
Comment (jzwart) [reply ci_handbook/modularity.qmd:9]: keeping as is

File: ci_handbook/data_format.qmd (hunk 4 above)
Comment (jzwart) [reply ci_handbook/data_format.qmd]: updated

File: ci_handbook/index.qmd (hunk 5 above)
Comment (jzwart) [reply ci_handbook/index.qmd:12]: Oh yeah, I think that would be it's own section / article. I added a new placeholder for that. Thanks!

File: ci_handbook/continuous_integration.qmd (hunk 6 above)
Comment (jzwart) [reply ci_handbook/continuous_integration.qmd:2]: This is referring to software development practice of continuous integration and continuous delivery, https://en.wikipedia.org/wiki/CI/CD ; there are various tools and best practices for how to go about this and I'm imagining having an article dedicated to this for forecast CI.  I'll leave as-is for now

**Pull request merged**